src/test/java/com/wikia/webdriver/common/remote/SiteId.java

```diff
@@ -0,0 +1,34 @@
+package com.wikia.webdriver.common.remote;
+
+
+import com.wikia.webdriver.common.contentpatterns.URLsContent;
+import com.wikia.webdriver.common.remote.operations.http.NoAuthOperation;
+import lombok.Getter;
+import org.json.JSONObject;
+
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class SiteId {
+
+  public SiteId(String wikiUrl) {
+    this.wikiUrl = wikiUrl;
+    this.extractSiteIdFromSpecialVersion();
+  }
+
+  @Getter
+  private String siteId;
+  private String wikiUrl;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Move this variable to comply with Java Code Conventions. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS1213)